src/akamai/js/akamaiHD_flash.js

```diff
@@ -1185,7 +1185,7 @@ var swfobject = function() {
       } else { // well-behaving browsers
         var o = createElement(OBJECT);
         o.setAttribute("type", FLASH_MIME_TYPE);
-        for (var m in attObj) {
+        for (var m = 0; m < attObj.length; m++) {
           if (attObj[m] != Object.prototype[m]) { // filter out prototype additions from other potential libraries
             if (m.toLowerCase() == "styleclass") { // 'class' is an ECMA4 reserved keyword
```

Comment:
This is not a current copy of the code. I don't know why github is showing the old version in here rather than the current.